zeeguu/core/feed_handler/feed_handler.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime, timezone, timedelta
+from zeeguu.core.util.time import normalize_to_server_time
+
+class FeedHandler():
+    def __init__(self, url:str, feed_type:int):
+        self.url = url
+        self.feed_type = feed_type
+        self.title = ""
+        self.description = ""
+        self.image_url_string = ""
+
+    def get_utc_time(self, article_date) -> datetime:
```

Comment:
ah. timezones again!
why is it called `get_utc_time` if it gets current time on the server? 
should it not be called `get_server_time`?

Reply:
Correct, it should be called get_server_time , I will update it.